js/src/components/paid-ads/campaign-form-content.scss

```diff
@@ -25,12 +25,4 @@
 		margin: 0;
 		font-style: italic;
 	}
-
-	.woocommerce-tree-select-control__label {
-		font-size: 13px;
-	}
-
-	.woocommerce-tree-select-control__help {
-		font-style: italic;
-	}
```

Comment:
Since the help prop in several control components of `@wordpress/components` and `@wocommerce/components` has not set the italic font style as default, I intended to set the italic here and remove the italic style from TreeSelectControl, but I missed that. 🤦‍♂️ 

Ref:
- The [base help style](https://github.com/WordPress/gutenberg/blob/%40wordpress/components%4012.0.8/packages/components/src/base-control/styles/base-control-styles.js#L29-L34) in `@wordpress/components` has no `italic` style.
- The [help of SelectControl](https://github.com/woocommerce/woocommerce-admin/blob/v2.7.2/packages/components/src/select-control/control.js#L254-L259) in `@wocommerce/components` doesn't have any default style.

Reply:
@eason9487 , thanks for the context, not a problem, it happens to everyone of us. 😄 I remember the italic style was specifically there for Google Listings and Ads, and WooCommerce Design Library, but I just check WC Design Library figma (LcsjAJaTHAQBgIzDfP9BtK-fi-3523%3A9319), apparently there is no more italic style for helper text. 🤔 

> I intended to set the italic here and remove the italic style from TreeSelectControl, but I missed that.

Should we do it in this PR and remove the italic style from TreeSelectControl? Right now we still have the italic style in TreeSelectControl (see screenshot in the PR description). I'm not sure if we should really do this, I remember seeing italic helper text in TreeSelectControl P2 and figma design, but couldn't find it now. The closest thing I find is Edit Paid Campaign figma design (https://www.figma.com/file/eQ9O4m2flzrcAiDMXkOW0m/Google-Listings-%26-Ads-v1.x?node-id=5519%3A115701).

Reply:
Currently, there are three places having the helper text under the selector:
1. Step 2 of onboarding setup
2. Step 1 of editing free listings
3. Edit a paid campaign.

But the first two didn't use TreeSelectControl's `help`. Therefore, if they are expected to be changed to use TreeSelectControl's `help` in the future, I think we could move the `italic` style to supported-country-select.scss. Otherwise, it should be ok to keep it here. And both ways remove the `italic` style from the tree-select-control/index.scss.

Reply:
> Therefore, if they are expected to be changed to use TreeSelectControl's help in the future, I think _we could move the italic style to supported-country-select.scss_. Otherwise, it should be ok to keep it here. And _both ways remove the italic style from the tree-select-control/index.scss_.

Okay, gotcha. Since the WordPress and WooCommerce components don't have the italic style, yeah I think it makes sense to remove the italic style from TreeSelectControl, and I think it is better to move it into SupportedCountrySelect. I'll look into this.

Reply:
I have made the changes in [`aab170e` (#1465)](https://github.com/woocommerce/google-listings-and-ads/pull/1465/commits/aab170e033f2ba5046ed0cef1f36c4446f60d483) and [`8e45a3e` (#1465)](https://github.com/woocommerce/google-listings-and-ads/pull/1465/commits/8e45a3e7d28c80524cb7a5769769dd3d0e992f5b). @eason9487 , would you like to have a review on this?

Reply:
> I have made the changes in [`aab170e` (#1465)](https://github.com/woocommerce/google-listings-and-ads/pull/1465/commits/aab170e033f2ba5046ed0cef1f36c4446f60d483) and [`8e45a3e` (#1465)](https://github.com/woocommerce/google-listings-and-ads/pull/1465/commits/8e45a3e7d28c80524cb7a5769769dd3d0e992f5b). @eason9487 , would you like to have a review on this?

The changes look good. Thanks for the help! 🙏